### PR TITLE
yordis/feat add env var for websocket host

### PR DESCRIPTION
**Pull request by yordis**

### CODE REVIEW

**Review by brooksmtownsend — COMMENTED** [review]
This looks fine to me, would like @lachieh's opinion just to make sure I'm not missing anything

File: packages/washboard/src/lattice/lattice-service.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ class LatticeService {
   private connection?: NatsConnection;
 
   private config = {
-    latticeUrl: 'ws://localhost:4001',
+    latticeUrl: import.meta.env.VITE_NATS_WEBSOCKET_URL,
```
Comment (lachieh) [inline packages/washboard/src/lattice/lattice-service.ts]: Let's include a default if the env is not set
```suggestion
    latticeUrl: import.meta.env.VITE_NATS_WEBSOCKET_URL ?? 'ws://localhost:4001',
```

**Review by lachieh — CHANGES_REQUESTED** [review]

File: packages/washboard/src/lattice/lattice-service.ts (hunk 1 above)
Comment (autodidaddict) [reply packages/washboard/src/lattice/lattice-service.ts]: You TypeScript people, always bragging about your `??` like you're so cool 😁

Comment (lachieh) [reply packages/washboard/src/lattice/lattice-service.ts]: I always love finding places to use the elvis operator `?:`, although in JS/TS it's `??=`

Comment (autodidaddict) [reply packages/washboard/src/lattice/lattice-service.ts]: Almost as cool as the Rust toilet bowl : `|_|()`

**Review by lachieh — APPROVED** [review]

**Review by connorsmith256 — APPROVED** [review]

**Pull request merged**